pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <!-- OpenMRS modules -->
     <fhir2Version>1.4.0-SNAPSHOT</fhir2Version>
+    <create.customer.if.not.exist>true</create.customer.if.not.exist>
```

Comment:
Same thing. Please put that in its own section in the props. This is certainly not an OpenMRS module.